Game/assets/shaders/fragment.glsl

```diff
@@ -84,40 +89,65 @@ in VertexData fragment;
 // Outputs
 out vec4 color;
 
-vec3 SchlickFresnel(const vec3 f0, float cos_theta)
+vec3 SchlickFresnel(const vec3 f0, float half_vector)
 {
-    return f0 + (vec3(1.0) - f0) * pow(1.0 - cos_theta, 5.0);
+    return f0 + (vec3(1.0) - f0) * pow(1.0 - half_vector, 5.0);
+}
+
+float GGX(const vec3 normal, const vec3 halfway_dir, float roughness)
+{
+    float a2 = pow(roughness, 2.0);
```

Comment:
in terms of maintainability, it would be better to use more descriptive variable names

Reply:
Thats how the variables are called in the equations. If you want instead of a2 and such it can be alpha_square, but if this functions are mantained it should be using the equations, and at this point I'm not sure which variable is quicker to identify.
![image](https://user-images.githubusercontent.com/85954068/166249913-d57704af-da56-40cc-9dd6-26dfe8a292c3.png)